client/my-sites/upgrades/checkout-thank-you/style.scss

```diff
@@ -145,31 +145,76 @@
 
 // Footer Subcomponent styles
 .checkout-thank-you__footer {
-	color: $gray;
+	padding: 32px;
+	text-align: left;
+
+	@include breakpoint( ">660px" ) {
+		padding-bottom: 48px;
+		padding-left: 24px;
+		padding-right: 24px;
+		padding-top: 48px;
+	}
 }
 
 .checkout-thank-you__footer-content {
-	margin: 16px auto;
+	margin-left: auto;
+	margin-right: auto;
```

Comment:
Because in that case you're also resetting top and bottom margins. Moreover when it's explicit like this you don't have to think about it (and my old brain likes that), and it will be compiled as a single property/line by SASS anyway.